### PR TITLE
Bugfix for Rogue Backfill Diff Not Updating Records

### PR DESCRIPTION
**Pull request by sheyd**
#### What's this PR do?
1. Added current_page =1 assignment when doing non-full backfill.
2. Full backfill from the top is state dependent on setting
Quasar DB status tracking page to 1, not entirely happy about
this, but we don't usually do full backfills, and it's a known
quantity.
#### Where should the reviewer start?
One file below.
#### How should this be manually tested?
Test locally and records will be updated.
#### What are the relevant tickets?
#396

### CODE REVIEW

File: quasar/campaign_activity.py [hunk 1]
```diff
@@ -23,17 +25,19 @@ def backfill_since():
...
                    headers={'X-DS-Rogue-API-Key': config.DS_ROGUE_API_KEY},
                    params={'page': 1, 'limit': 40, 'filter[updated_at]': start_time})
     final_page = rogueAPI.get('')['meta']['pagination']['total_pages']
-    current_page = _get_start_page(db)
+
+    if hours is not None:
+        print("Current backfill hours are %s." % hours)
+        current_page = 1
+    else:
+        current_page = _get_start_page(db)
```
Comment (xianny) [inline quasar/campaign_activity.py]: Not sure I understand what's going on here. 

Sounds like:

If we are backfilling x hours, set current page to 1

If we're doing full backfill, then get the start page from the db.

Is that right?

File: quasar/campaign_activity.py [hunk 2]
```diff
@@ -15,6 +15,8 @@
 
 def full_backfill():
     _backfill()
+    # There is an implicit state dependency here that requires
+    # setting Quasar DB page_progress to 1 for full backfill restart.
```
Comment (xianny) [inline quasar/campaign_activity.py]: I'm a bit confused by this comment here. Could this be a docstring? E.g. "Processes Rogue data starting from last processed page recorded in ROGUE_PROGRESS_TABLE; to restart, we have to manually set page_progress to 1".

File: quasar/campaign_activity.py (hunk 1 above)
Comment (sheyd) [inline quasar/campaign_activity.py]: @xianny Yes, that's exactly what's going on!

File: quasar/campaign_activity.py (hunk 2 above)
Comment (sheyd) [inline quasar/campaign_activity.py]: You're right, I'll update this.

**Pull request merged**